### DISCUSSION TITLE
[Docs] [carousel] 文档属性名写错了

### DISCUSSION THREAD

**Discussion by star-hs** [element-plus/element-plus#12502] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `carousel`

## Description

文档中提到的referrer-policy 实际应该是referrerpolicy属性，看了源码后发现实际element对这个属性并没有做特殊处理，只是透传的。
那就不要写短横线，导致实际没生效

### Additional comments

_(empty)_

![image](https://user-images.githubusercontent.com/44965163/232694583-f87308bc-3f6f-43e9-8db8-56792edba98d.png)

**Comment by chenxch** [comment]
#12503

**Comment by chenxch** [comment]
Thanks again for the feedback.
